test/hotspot/jtreg/vmTestbase/gc/gctests/ReferencesGC/ReferencesGC.java

```diff
@@ -134,12 +131,12 @@ public void run() {
                     holder[i] = null;
                 }
 
-                Algorithms.eatMemory(stresser);
+                // WB.fullGC() is a lst ditch GC which should provoke
```

Comment:
```suggestion
                // WB.fullGC() is a last ditch GC which should provoke
```

Reply:
Maybe just write. `WB.fullGC()` is guaranteed to clear all kinds of weak references.